src/main/java/com/lazerycode/jmeter/mojo/AbstractJMeterMojo.java

```diff
@@ -271,4 +278,12 @@ protected void loadMavenProxy() {
 
 	}
 
```

Comment:
the root cause of the problem is that this code needs to be executed by both the `ConfigureMojo` and the `RunJMeterMojo` / `RunJMeterGUIMojo`. (see description in #195)
If the code stays in the `ConfigureMojo`, it will still not be possible to change the format of the output. ;)